decrypt_oracle/src/aws_encryption_sdk_decrypt_oracle/app.py

```diff
@@ -59,7 +60,7 @@ def basic_decrypt() -> Response:
     APP.log.debug(APP.current_request.raw_body)
 
     try:
-        client = aws_encryption_sdk.EncryptionSDKClient()
+        client = aws_encryption_sdk.EncryptionSDKClient(commitment_policy=CommitmentPolicy.FORBID_ENCRYPT_ALLOW_DECRYPT)
```

Comment:
Maybe a comment about why this is the right policy.
When I read it the first time I had to think about it a bit :)
What do you think about:

```suggestion
        // The decrypt oracle needs to be able to decrypt any message
        // it does not encrypt messages for anyone.
        client = aws_encryption_sdk.EncryptionSDKClient(commitment_policy=CommitmentPolicy.FORBID_ENCRYPT_ALLOW_DECRYPT)
```